src/Apps/Order/OrderApp.tsx

```diff
@@ -33,6 +45,21 @@ export class OrderApp extends React.Component<OrderAppProps> {
     )
   }
 
+  componentDidMount() {
+    if (window.Stripe) {
+      this.setState({
+        stripe: window.Stripe("pk_test_BGUg8FPmcBs1ISbN25iCp2Ga"),
```

Comment:
We should use an ENV var for this (or `sd`, whatever the current pattern is). If you don't want to add it to this PR please add a TODO 😄